src/main/java/fibonacci/Fibonacci.java

```diff
@@ -22,7 +29,17 @@ else if(number == 1){
         else{
             rval = fib(number - 1) + fib(number - 2);
         }
-
+        fibNumber = rval;
```

Comment:
Why was this done?  It resets a class variable over and over again when the method just returns the value.